PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java

```diff
@@ -81,6 +82,7 @@
     protected JLabel storageRootPathLabel;
     protected JComboBox sparkVersionComboBox;
     protected JLabel sparkVersionLabel;
+    private JXHyperLinkWithUri jobQueueHyperLink;
```

Comment:
`@Nullable`

Reply:
We can `setURI(null)` to `jobQueueHyperLink`, but the field itself should not be null

---
In reply to: [209217336](https://github.com/Microsoft/azure-tools-for-java/pull/1872#discussion_r209217336) [](ancestors = 209217336)